applications/picocdn-node-vm/README.md

```diff
@@ -0,0 +1,87 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 NetPico LAbs Pvt Ltd.(picoNETS)
+```
+
+
+# **picoCDN Platform Overview**
+picoCDN is a Deep Edge focused Caching and Content Delivery Network (CDN) platform. This Deep Edge CDN platform performs better than traditional CDNs by virtue of it's unique and flexible architecture. This is acheived by bringing the content closer to the users(from a network, rather than geographical perspective), by embedding deep into Telco and ISP networks.  With this Ultra-Low Latency CDN platform, ISPs/CSPs/Telcos can deliver a buffer-free experience for Full HD, 4K, AR, VR, and other bandwidth-demanding content. Every edge location of the distribution network is geared to deliver content to users who are just a couple of hops away. picoCDN also supports a pre-caching API to push content to the edge nodes before customers requests reach the node. This version of the platform is designed to help OEMs/ODMs/ISPs/CSPs/Telcos deploy Edge POPs in various local ISPs, malls, hotels, airports, cellsites or other 5G Multi-Access Edge Computing (MEC) locations running Intel OpenNESS.
+
+
+## **Pre Requisites – Resources Required**
+
+| **picoCDN-VM Variants**       | **(S) Small**        | **(M) Medium**       | **(L) Large**        |
+| ----------------------------- | -------------------- | -------------------- | -------------------- |
+| **Service Performance**       |                      |                      |                      |
+| Content Delivery              | 1  Gbps              | 5  Gbps              | 10  Gbps             |
+| **Required Resources**        | -------------------- | -------------------- | -------------------- |
+| Compute* (vCores)             | 4-8                  | 12+                  | 24+                  |
+| Memory (RAM)                  | 16 GB                | 64 GB                | 128 GB               |
+| Storage (System+Logs minimum) | 100  GB              | 200  GB              | 500  GB              |
+| Storage (Content recommended) | 2-5 TB SSD           | 10-15 TB SSD         | 15-30 TB SSD         |
+| Storage Performance           | 1.5 GBps  800 IOPS   | 6 GBps   1600 IOPS   | 12 GBps    3200 IOPS |
+| Host OS                       <td colspan=3> picoCDN7 Custom Image </td></tr>
+| Network                       <td colspan=3> 1G / 10G / 25G+ via  Virtio / Direct-Attach / SR-IOV</td></tr>
+
+
+## **Where to Purchase**
+
+The picoCDN custom image can be obtained from the picoNETS support portal. For more information visit [www.piconets.com](https://www.piconets.com) or contact us via email on info@piconets.com
+
+
+## Pre Requisites - Installing OpenNESS
+
+|**Configuration**                   |                       |
+|  ----------------------------------|-----------------------|
+| OpenNESS Version                   | 20.06                 |
+| Flavor Used 			     | kubevirt VM           |
+| Distribution	                     | IDO       	     |

```

Comment:
Can you please change the distribution to Opensource.

Reply:
Done.